ultrasonic/src/main/kotlin/org/moire/ultrasonic/fragment/PlayerFragment.kt

```diff
@@ -979,8 +982,34 @@ class PlayerFragment :
         if (currentPlaying != null) {
             currentSong = currentPlaying!!.song
             songTitleTextView.text = currentSong!!.title
-            albumTextView.text = currentSong!!.album
             artistTextView.text = currentSong!!.artist
+            albumTextView.text = currentSong!!.album
+            if (currentSong!!.year != null && Settings.showNowPlayingDetails)
+                albumTextView.append(String.format(Locale.ROOT, " (%d)", currentSong!!.year))
+
+            if (Settings.showNowPlayingDetails) {
+                genreTextView.text = currentSong!!.genre
+                genreTextView.visibility =
```

Comment:
Shorter syntax:  .isVisible = true/false

Reply:
Tx for the hint! Done.